src/Stratis.Bitcoin.Tests.Common/Transaction.cs

```diff
@@ -0,0 +1,20 @@
+﻿using FluentAssertions;
+using NBitcoin;
+
+namespace Stratis.Bitcoin.Tests.Common
+{
+    public class Transactions
+    {
+        public static Transaction BuildNewTransactionFromExistingTransaction(Transaction inputTransaction, int index = 0)
+        {
+            var transaction = new Transaction();
```

Comment:
There is a property on transaction called Clone is that not better?

Reply:
I think this one takes an existing transaction to use as inputs, not really cloning it...